Gems/EMotionFX/Code/Source/Integration/System/SystemComponent.cpp

```diff
@@ -564,13 +564,12 @@ namespace EMotionFX
         //////////////////////////////////////////////////////////////////////////
         void SystemComponent::EnableRayRequests()
         {
-            RaycastRequestBus::Handler::BusDisconnect();
-            RaycastRequestBus::Handler::BusConnect();
+            AZ::Interface<IRaycastRequests>::Register(this);
```

Comment:
question: Does `Register` gracefully handle being called twice without a disconnect? (as there was a disconnect then connect before it just made me wonder)